common/policydsl/policydsl_builder.go

```diff
@@ -4,15 +4,14 @@ Copyright IBM Corp. 2016 All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package cauthdsl
+package policydsl
```

Comment:
nit: the file name stutters.

it stuttered before and it still stutters. if you think of a better name, feel free to update it in a separate pr.